doc/usecases/snippet_sharing/UC_edit_snippet.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Edit configuration snippet
+
+## Summary
+
+Title: Edit configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user (snippet owner), Moderator
+Brief: Users can edit their own database entries. Moderators can edit all configuration snippets in the database, also the ones of other users.

```

Comment:
No storing of different versions. But you can create a new snippet from the old one (in the GUI), which you could use to build a history if you really need it.

Help system in what sense? Some explanations of what is what and how it works should obviously be part of the web GUI later for sure. But that's not worth being mentioned in the use cases.